### DISCUSSION TITLE
The calendar of what country is 7-8 years behind the standard Gregorian calendar used in most of the world?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#752] [Galaxy Trivia] [answered]
*answer with country emoji*

**Comment by SANKARAMDAS** [comment]
⠀⠀⠀⠀⠀⢀⣠⣶⠶⠾⠿⠛⠛⠿⠷⣶⣶⣄⡀⠀⠀⠀⠀⠀
⠀⠀⠀⣠⡾⠟⠉⠀⠀⢀⣠⣤⣶⠾⠟⠛⠛⠉⠻⢷⣄⠀⠀⠀
⠀⢀⣾⣟⣀⣀⣠⣴⠿⠛⠉⠁⠀⠀⢀⣀⣠⣤⣤⣤⣽⣷⡀⠀
⠀⣾⠋⠉⠙⠛⠛⠛⠛⠿⠷⠶⢶⣿⣿⣋⣉⠉⠉⠉⠉⠙⣷⠀
⣸⡏⠀⠀⠀⠀⠀⠀⣀⣀⣀⣀⣀⣀⣉⠉⠛⠻⣶⣦⣄⡀⢹⣇
⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿
⣿⣿⣿⣿⡙⠛⠿⢿⣿⣿⣿⣿⣿⣿⣿⣿⡿⠿⠛⢋⣿⣿⣿⣿
⣿⣿⣿⣿⣷⣀⠀⠀⢀⣹⣿⣿⣿⣿⣏⡀⠀⠀⣀⣾⣿⣿⣿⣿
⢸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡇
⠀⣿⡍⠉⠛⠻⠿⣿⣿⣿⣛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⢻⣿⠀
⠀⠘⣿⣦⣀⠀⠀⠀⠈⠉⠛⠻⠿⠷⢶⣤⣤⣄⣀⣀⣀⣾⠃⠀
⠀⠀⠘⣿⡙⠿⣶⣤⣀⠀⠀⠀⠀⠀⠀⠀⠉⠉⠉⢉⣿⠃⠀⠀
⠀⠀⠀⠘⢷⣄⠀⠉⠛⠻⠷⣶⣤⣤⣀⣀⡀⠀⣠⡾⠃⠀⠀⠀
⠀⠀⠀⠀⠀⠙⢷⣄⡀⠀⠀⠀⠀⠈⠉⢛⣻⡿⠋⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠉⠻⠷⢶⣦⣴⡶⠾⠟⠉⠀⠀⠀⠀⠀⠀⠀

An 🇪🇹-Ethiopian year is comprised of 13 months, and is seven years behind the Gregorian calendar. In fact, Ethiopians celebrated the new millennium on September 11, 2007; this is because the Ethiopians continued with the same calendar that the Roman church amended in 525 AD.

**Comment by peterbaccaro** [comment, marked as answer]
🇪🇹 - The calendar of Ethiopia is approximately 7-8 years behind the standard Gregorian calendar used in most of the world. Ethiopia uses a calendar that is based on the Coptic calendar, which is a solar calendar that is similar to the Julian calendar. The Ethiopian calendar has 12 months of 30 days each, and a 13th month of 5 or 6 days depending on whether it is a leap year or not. The Ethiopian New Year falls on September 11 or 12 in the Gregorian calendar, depending on whether it is a leap year or not. The Ethiopian calendar is still widely used in Ethiopia for religious and cultural events.
